rebasehelper/tags.py

```diff
@@ -85,20 +108,23 @@ def sanitize(name):
                 return '{0}{1}'.format(tokens[0], int(tokens[1]))
             return name.capitalize()
         result = []
+        parsed_mapping = cls._map_sections_to_parsed(raw_content, parsed_content)
         tag_re = re.compile(r'^(?P<prefix>(?P<name>\w+)\s*:\s*)(?P<value>.+)$')
-        for section, _ in raw_content.sections:
+        for section_index, (section, section_content) in enumerate(raw_content.sections):
             section = section.lower()
             if not section.startswith('%package'):
                 continue
-            parsed = parsed_content.section(section)
-            for index, line in enumerate(raw_content.section(section)):
+            parsed_index = parsed_mapping[section_index]
+            parsed: List[str] = [] if parsed_index == -1 else parsed_content.sections[parsed_index][1]
+            for index, line in enumerate(section_content):
```

Comment:
This is more of a nitpick, but there are 3 index variables already, so using just `index` is not very clear. `line_index` would be better. And `parsed_index` should probably be `parsed_section_index`. I don't like long variable names but I think clarity matters more.

Reply:
This comment is still valid.